.gitattributes

```diff
@@ -0,0 +1,5 @@
+
+###############################################################################
+# Set default behavior to automatically normalize line endings.
+###############################################################################
+* text=auto
```

Comment:
```suggestion
* text=auto

# Force bash scripts to always use lf line endings so that if a repo is accessed
# in Unix via a file share from Windows, the scripts will work.
*.in text eol=lf
*.sh text eol=lf

# Likewise, force cmd and batch scripts to always use crlf
*.cmd text eol=crlf
*.bat text eol=crlf

*.cs text=auto diff=csharp
*.csproj text=auto
*.sln text=auto
*.resx text=auto
*.xml text=auto
*.txt text=auto
```